src/components/Header/HeaderContact.js

```diff
@@ -82,7 +82,6 @@ const styles = StyleSheet.create({
   container: {
     flexDirection: 'row',
     justifyContent: 'space-between',
-    paddingVertical: SIZES.base,
     alignItems: 'center'
   },
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `paddingVertical` style property has been removed from the container style. This could potentially affect the layout of your component, especially if it was used to provide spacing above and below the container. If this change was intentional and you've verified that it doesn't negatively impact your layout, then it's fine. Otherwise, you might want to reconsider this change.
```diff
   container: {
     flexDirection: 'row',
     justifyContent: 'space-between',
+    paddingVertical: SIZES.base,
     alignItems: 'center'
   },
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->